aries-site/src/examples/templates/dashboards/content/chartCards/CostCompact.js

```diff
@@ -0,0 +1,181 @@
+            });
+          }
+        }
+      });
+      nextValues
+        .sort((a, b) => b.value - a.value)
+        .forEach((value, index) => {
+          // eslint-disable-next-line no-param-reassign
+          value.color = chartColors[index % chartColors.length];
+        });
+      setValues(nextValues);
+    }
+  }, [chartColors, consumptionData, reportWindow.begin, reportWindow.end]);
+
+  // Calculate total cost
+  useEffect(() => {
+    let nextTotalCost = 0;
+    if (values) {
+      Object.keys(values).forEach(key => {
+        nextTotalCost += values[key].value;
+      });
+      setTotalCost(nextTotalCost);
+    }
+  }, [values]);
+
+  const grid = {
+    columns: {
+      xsmall: [['xsmall', 'small'], 'auto'],
+      small: [['xsmall', 'small'], 'auto'],
+      medium: ['auto', 'auto'],
+      large: ['auto', 'auto'],
+      xlarge: ['auto', 'auto'],
+    },
+    rows: [['auto']],
+    areas: {
+      xsmall: [['chart', 'legend']],
+      small: [['chart', 'legend']],
+      medium: [['chart', 'legend']],
+      large: [['chart', 'legend']],
+      xlarge: [['chart', 'legend']],
+    },
+    gap: {
+      xsmall: 'medium',
+      small: 'medium',
+      medium: 'small',
+      large: 'medium',
+      xlarge: 'medium',
+    },
+  };
+
+  return (
+    <ChartCard title="Cost" subtitle="Consumption Analytics" gap="medium">
+      {values && (
+        <Grid
+          columns={grid.columns[size]}
+          rows={grid.rows}
+          areas={grid.areas[size]}
+          gap={grid.gap[size]}
+        >
+          <Stack alignSelf="start" gridArea="chart" anchor="center">
+            <Box
+              alignSelf="start"
+              height={{
+                max: `${parseMetricToNum(theme.global.size.small) +
+                  parseMetricToNum(theme.global.size.xsmall)}px`,
+              }}
+            >
+              <Meter
+                values={values.map((value, index) => ({
+                  label: value.label,
+                  value: value.value,
+                  color: {
+                    color: chartColors[index % chartColors.length],
+                    opacity:
+                      !active || (active && active === value) ? '100%' : 'weak',
+                  },
+                }))}
+                size="full"
+                type="circle"
+              />
+            </Box>
+            <Measure
+              alignSelf="center"
+              align="center"
+              direction="column-reverse"
+              name={{ label: { label: period, size: 'xsmall' } }}
+              value={{
+                value: formatCurrency(active ? active.value : totalCost),
+                size: 'xlarge',
+              }}
+              justify="center"
+              width="xsmall"
+            />
+          </Stack>
+          <Legend
+            alignSelf="center"
+            gridArea="legend"
+            mode="compact"
+            onClick={datum => setActive(active !== datum ? datum : undefined)}
```

Comment:
To create the tie/reference between values prop.

```suggestion
            onClick={value => setActive(active !== value ? value : undefined)}
```